charts/stable/cloudreve/values.yaml

```
@@ -1,32 +1,58 @@
 image:
   pullPolicy: IfNotPresent
-  repository: xavierniu/cloudreve
-  tag: latest@sha256:cdd00367bbdf3247e15feebfbde354237586a308c6fac730856670dd31b44b47
+  repository: cloudreve/cloudreve
+  tag: 3.8.3@sha256:a9e2373b7ca59bc43b9a4ceb50e7f6b80539c1732a9439b96dfe1aec6354a6c5
 persistence:
   avatarpath:
     enabled: true
     mountPath: /cloudreve/avatar
   config:
     enabled: true
-    mountPath: /cloudreve/config
-  databasepath:
-    enabled: true
-    mountPath: /cloudreve/db
+    targetSelector:
+      main:
+        main:
+          mountPath: /cloudreve
+        init:
+          mountPath: /conf-init
   uploadpath:
     enabled: true
     mountPath: /cloudreve/uploads
-portal:
-  open:
-    enabled: true
 securityContext:
   container:
     readOnlyRootFilesystem: false
     runAsGroup: 0
     runAsUser: 0
+workload:
+  main:
+    podSpec:
+      initContainers:
+        init:
+          enabled: true
```

```suggestion
          enabled: "{{ or .Release.IsUpgrade .Release.IsInstall }}"
```

It **should** work if I remember correctly :D